app/components/ClientSampleCode.tsx

```diff
@@ -403,5 +403,72 @@ export function ClientSampleCode({
           <Highlight language="sh" code="./a.out" />
         </>
       )
+    case 'cs':
+      return (
+        <>
+          In Visual Studio, create a new C# Console App under File {'>'} New{' '}
+          {'>'} Project. After the project initializes, right click the solution
+          in the Solution Explorer and click Manage NuGet packages for solution.
+          Browse for and install Confluent.Kafka. Copy the following into your
+          Progam.cs file.
+          <Highlight
+            language="cs"
+            filename={`example.${language}`}
+            code={dedent(String.raw`
+            using Confluent.Kafka;
+
+
+            var config = new ConsumerConfig
+            {
+              SecurityProtocol = SecurityProtocol.SaslSsl,
+              BootstrapServers = "kafka.${domain ?? 'gcn.nasa.gov'}",
+              GroupId = Guid.NewGuid().ToString(),
+              SaslMechanism = SaslMechanism.OAuthBearer,
+              SaslOauthbearerMethod = SaslOauthbearerMethod.Oidc,
+              SaslOauthbearerTokenEndpointUrl = "https://auth.${
+                domain ?? 'gcn.nasa.gov'
+              }/oauth2/token",
+              // Warning: don't share the client secret with others
+              SaslOauthbearerClientId = "${clientId}",
+              SaslOauthbearerClientSecret = "${clientSecret}"
+            };
+
+            // Create a client.
+            using (var consumer = new ConsumerBuilder<Ignore, string>(config).Build())
+            {
+              // Subscribe to topics
+              consumer.Subscribe(new List<string> {
+                ${topics.map((topic) => `"${topic}"`).join(`,
+                `)}
+              });
+
+              // List all topics
+              consumer.Subscription.ForEach(topic => Console.WriteLine(topic));
```

Comment:
Should only be displayed if `listTopics` is true.

Reply:
Ping! Would you please wrap this up?

Reply:
Updated in latest commit